src/main/scala/org/allenai/plugins/VersionInjectorPlugin.scala

```diff
@@ -117,6 +136,51 @@ object VersionInjectorPlugin extends AutoPlugin {
         gitConfFile
     }
 
+  val injectCacheKeyTask = injectCacheKey := {
+    val resourceManagedPath = (resourceManaged in Compile).value.absolutePath
+    val org = organization.value
+    val cleanName = cleanArtifactName(name.value)
+    val logger = streams.value.log
+    val cacheKeyResult = cacheKey.value
+    val cacheKeyConfFile = new java.io.File(s"$resourceManagedPath/$org/$cleanName/cacheKey.conf")
+    
+    logger.info(s"Generating cacheKey.conf managed resource... (cacheKey: $cacheKeyResult)")
+
+    val cacheKeyContents = s"""cacheKey: "$cacheKeyResult""""
+    IO.write(cacheKeyConfFile, cacheKeyContents)
+    cacheKeyConfFile
+  }
+
+  //Gives us the git most recent commits for all the local projects that this project depends on
+  // We have to use a dynamic task because generating tasks based on project dependencies
+  lazy val dependentGitCommits: Def.Initialize[Task[Seq[String]] = Def.taskDyn {
+    // necessarily alters the task graph
```

Comment:
Move up to the first half of this comment.

Also, you should terminate your sentences with a period.